features/text_components/publication_status.feature

```diff
@@ -0,0 +1,27 @@
+@311
+Feature: Only published text components should used to generate a report
+  As a reporter
+  I want to have a flag for text.components (draft, fact-checked, published) and only published text components should be included in the report
+  In order to add new text components without affecting the result of the live report
+
+  Background:
+    Given there is a sensor live report
+  
+  Scenario:
```

Comment:
This scenario is perfectly valid. But how about describing that the journalist *changes* the `publication_status` and then suddenly the text component is visible in the text? Is it better, ie. more descriptive?